operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/index.tsx

```diff
@@ -37,14 +39,23 @@ const VariablePanel = observer(function VariablePanel() {
   }, [processInstanceId]);
 
   useEffect(() => {
-    if (flowNodeId) {
+    if (!flowNodeInstanceId && flowNodeId) {
       fetchListeners({
         fetchType: 'initial',
         processInstanceId: processInstanceId,
         payload: {flowNodeId},
       });
+    } else if (
+      flowNodeInstanceId &&
+      !flowNodeSelectionStore.isRootNodeSelected
```

Comment:
By default the selection store returns the process instance id as the `flowNodeInstanceId`, this is to avoid a request being made when entering the screen since the root node will never have listeners attached to it.